PetaPoco.Tests.Integration/Databases/MSAccess/MsAccessQueryTests.cs

```diff
@@ -171,6 +171,199 @@ public override void Query_ForMultiPocoWithPropertyMissingSetMethod_ShouldThrow(
+        [Fact]
+        public override void QueryMultiple_ForSingleResultsSetWithMultiPoco_ShouldReturnValidPocoCollection()
+        {
+            AddJoinableOrders(1);
+
+            var pd = PocoData.ForType(typeof(JoinablePerson), DB.DefaultMapper);
+            var od = PocoData.ForType(typeof(JoinableOrder), DB.DefaultMapper);
+            var pdId = pd.Columns.Values.First(c => c.PropertyInfo.Name == "Id").ColumnName;
+            var pdName = pd.Columns.Values.First(c => c.PropertyInfo.Name == "Name").ColumnName;
+            var odPersonId = od.Columns.Values.First(c => c.PropertyInfo.Name == "JoinablePersonId").ColumnName;
+
+            /*
+            "SELECT TOP 1 *\r\n                         FROM [JoinableOrders] o\r\n                         INNER JOIN [JoinablePeople] p ON p.[Id] = o.[JoinablePersonId]\r\n                         WHERE p.[FullName] = @0\r\n                         ORDER BY 1 DESC;"
+
+            SELECT TOP 1 *
+                         FROM [JoinableOrders] o
+                         INNER JOIN [JoinablePeople] p ON p.[Id] = o.[JoinablePersonId]
+                         WHERE p.[FullName] = @0
+                         ORDER BY 1 DESC;
+            */
+
+            var sql = $@"SELECT TOP 1 *
+                         FROM {DB.Provider.EscapeTableName(od.TableInfo.TableName)} o
+                         INNER JOIN {DB.Provider.EscapeTableName(pd.TableInfo.TableName)} p ON p.{DB.Provider.EscapeSqlIdentifier(pdId)} = o.{DB.Provider.EscapeSqlIdentifier(odPersonId)}
+                         WHERE p.{DB.Provider.EscapeSqlIdentifier(pdName)} = @0
+                         ORDER BY 1 DESC;";
+
+            List<JoinableOrder> result;
+            using (var multi = DB.QueryMultiple(sql, "Peta0"))
+            {
+                result = multi.Read<JoinableOrder, JoinablePerson, JoinableOrder>((o, p) =>
+                {
+                    o.JoinablePerson = p;
+                    return o;
+                }).ToList();
+            }
+
+            result.Count.ShouldBe(1);
+
+            var order = result.First();
+
+            order.PoNumber.ShouldStartWith("PO");
+            order.Status.ShouldBeOneOf(Enum.GetValues(typeof(OrderStatus)).Cast<OrderStatus>().ToArray());
+            order.JoinablePersonId.ShouldNotBe(0);
+            order.CreatedOn.ShouldBeLessThanOrEqualTo(new DateTime(1990, 1, 1, 0, 0, 0, DateTimeKind.Utc));
+            order.CreatedBy.ShouldStartWith("Harry");
+
+            order.JoinablePerson.ShouldNotBeNull();
+            order.JoinablePerson.Id.ShouldNotBe(0);
+            order.JoinablePerson.Name.ShouldStartWith("Peta");
+            order.JoinablePerson.Age.ShouldBe(18);
+        }
+
+        // FIXME: System.Data.OleDb.OleDbException : Characters found after end of SQL statement.
+        [Fact]
+        public override void QueryMultiple_ForMultiResultsSetWithSinglePoco_ShouldReturnValidPocoCollection()
```

Comment:
Not sure what the issue is here...generated SQL is:
```sql
SELECT * FROM [Orders] o
WHERE o.[Id] = @0;
SELECT * FROM [People] p
WHERE p.[FullName] = @1;
```

Reply:
Following @Curlack's [comment](https://github.com/CollaboratingPlatypus/PetaPoco/pull/696#issuecomment-1737835299), and some [follow-up](https://stackoverflow.com/questions/1280021/in-ms-access-is-it-possible-to-execute-multiple-queries) [research](https://stackoverflow.com/questions/770614/running-multiple-sql-statements-in-the-one-operation), seems MS Access doesn't accept multiple statements in a single executed command in the context of standalone SQL queries (as supposed to VBA procedure or macro).